core/workspace_svg.js

```diff
@@ -286,24 +285,34 @@ Blockly.WorkspaceSvg.prototype.toolboxCategoryCallbacks_ = {};
  * @type {SVGMatrix}
  * @private
  */
-Blockly.WorkspaceSvg.prototype.inverseScreenCTM_ = null;
+Blockly.WorkspaceSvg.prototype.inverseScreenCTM_ = "dirty";
```

Comment:
Still make it a constant.  Add it at the end here: https://github.com/LLK/scratch-blocks/blob/develop/core/constants.js#L368